docs(renovate): enhance renovate documentation (#2199)
AmadeusITGroup/otter: docs/renovate/readme.md

> [!NOTE]
> The Otter Renovate presets are always prefixed with `github>AmadeusITGroup/otter//tools/renovate/` to target the specific configuration files hosted in the Otter repository.

> [!WARNING]

> > Check the *Self hosted Renovate section* below to see how to fully benefit from the presets.

Reviewer: ```suggestion
> Check the [Self hosted Renovate section](#self-hosted-renovate) to see how to fully benefit from the presets.
```

> > Check the [Self hosted Renovate section](#self-hosted-renovate) to see how to fully benefit from the presets.

Reviewer 2: ```suggestion
> Check out the [Self-Hosted Renovate section](#self-hosted-renovate) to see how to fully benefit from the presets.
```

## Available presets

> You can see here [the documentation of the available presets](https://github.com/AmadeusITGroup/otter/tree/main/tools/renovate).

Reviewer 2: ```suggestion
You can have a look at [the documentation of the available presets](https://github.com/AmadeusITGroup/otter/tree/main/tools/renovate).
```
(avoid using "here" in documentation)

## Self-Hosted Renovate

> The otter presets come with post upgrade commands to help the upgrade process by running migration scripts or regenerating the SDK for example.

Reviewer 3: ```suggestion
The Otter presets come with post upgrade commands to help the upgrade process by running migration scripts or regenerating the SDK for example.
```

This feature can only work if these commands are whitelisted.
Unfortunately, as of today, whitelisting of post upgrade commands is only possible when using a [Self-Hosted Renovate](https://docs.renovatebot.com/examples/self-hosting/).

Instead of using the GitHub App Renovate, you'll need to create your own or to create a workflow to run it.
The workflow can be as simple as a cron running `npx renovate`.

> You will need the following config in your renovate instance to support the otter presets:

Reviewer 3: ```suggestion
You will need the following config in your renovate instance to support the Otter presets:
```
